cli/main.go

```diff
@@ -481,6 +481,7 @@ func main() {
 			{Name: "Yuxiang Zhang", Email: "yuxiang.zhang@tuna.tsinghua.edu.cn"},
 			{Name: "Nogeek", Email: "ritou11@gmail.com"},
 			{Name: "ZenithalHourlyRate", Email: "i@zenithal.me"},
+			{Name: "Jiajie Chen", Email: "c@jia.je"},
```

Comment:
Scheduled:
It seems that OceanShen forgot to add himself/herself in older commits, hence we can add him/her here.

He/She provided the following information in privite communication.
```
KomeijiOcean <sa17@mails.tsinghua.edu.cn>
```

Reply:
Using a private mail address `oceans2000@126.com` might be more appropriate. But I don't think it is a good idea to add my name in an unrelated PR.